nestkernel/mpi_manager.cpp

```diff
@@ -251,9 +249,16 @@ nest::MPIManager::mpi_finalize( int exitcode )
       mpi_abort( exitcode );
     }
   }
+}
+
 #endif /* #ifdef HAVE_MPI */
```

Comment:
Doesn't this need to be an else, wit the endif following on line 261?
```suggestion
#else /* #ifdef HAVE_MPI */
```

Reply:
Good catch. Stupid copy&paste.